src/Components/EmailNotificationForm/EmailNotificationForm.js

```diff
@@ -82,12 +108,12 @@ class EmailNotificationForm extends Component {
   }
 
   clearInputs = () => {
-    this.setState({ 
-      first_name: "",
-      last_name: "",
-      state_name: "Select",
-      email: "",
-      language: "",
+    this.setState({
+      first_name: '',
+      last_name: '',
+      state_name: 'Select',
+      email: '',
+      language: '',
       agree_to_emails: false
     });
   }
```

Comment:
We may also want to think about how the error and successMessage properties will need to be reset.